chains/v2/chains.json

```diff
@@ -315,7 +315,7 @@
             },
             {
                 "assetId": 1,
-                "symbol": "kUSD",
+                "symbol": "KUSD",
```

Comment:
It was right before :D kUSD is the right name according to Karura branding

Reply:
fixed
thanks!